plugins/ingest-attachment/build.gradle

```diff
@@ -50,14 +50,24 @@ dependencies {
   api "org.apache.tika:tika-parsers:${versions.tika}"
   // Required for the various document parsers
   api "org.apache.tika:tika-parsers-standard-package:${versions.tika}"
+  api "org.apache.tika:tika-parser-apple-module:${versions.tika}"
+  api "org.apache.tika:tika-parser-html-module:${versions.tika}"
+  api "org.apache.tika:tika-parser-microsoft-module:${versions.tika}"
```

Comment:
@sandeshkr419 why do we need all these Tika parsers? We didn't bundle them before

Reply:
Yes, the tika parsers were not being packaged from tika 2.8 onwards.
https://github.com/apache/tika/pull/1130

I had to manually add them to resolve the below dependency failures after upgrading to tika 2.8+. 

```
/Users/--/workplace/opensearch/OpenSearch/plugins/ingest-attachment/src/main/java/org/opensearch/ingest/attachment/TikaImpl.java:94: error: package org.apache.tika.parser.html does not exist
        new org.apache.tika.parser.html.HtmlParser(),
                                       ^
/Users/--/workplace/opensearch/OpenSearch/plugins/ingest-attachment/src/main/java/org/opensearch/ingest/attachment/TikaImpl.java:95: error: package org.apache.tika.parser.pdf does not exist
        new org.apache.tika.parser.pdf.PDFParser(),
                                      ^
/Users/--/workplace/opensearch/OpenSearch/plugins/ingest-attachment/src/main/java/org/opensearch/ingest/attachment/TikaImpl.java:96: error: package org.apache.tika.parser.txt does not exist
        new org.apache.tika.parser.txt.TXTParser(),
                                      ^
/Users/--/workplace/opensearch/OpenSearch/plugins/ingest-attachment/src/main/java/org/opensearch/ingest/attachment/TikaImpl.java:97: error: package org.apache.tika.parser.microsoft.rtf does not exist
        new org.apache.tika.parser.microsoft.rtf.RTFParser(),
                                                ^
/Users/--/workplace/opensearch/OpenSearch/plugins/ingest-attachment/src/main/java/org/opensearch/ingest/attachment/TikaImpl.java:98: error: package org.apache.tika.parser.microsoft does not exist
        new org.apache.tika.parser.microsoft.OfficeParser(),
                                            ^
/Users/--/workplace/opensearch/OpenSearch/plugins/ingest-attachment/src/main/java/org/opensearch/ingest/attachment/TikaImpl.java:99: error: package org.apache.tika.parser.microsoft does not exist
        new org.apache.tika.parser.microsoft.OldExcelParser(),
                                            ^
/Users/kusandes/workplace/opensearch/OpenSearch/plugins/ingest-attachment/src/main/java/org/opensearch/ingest/attachment/TikaImpl.java:100: error: package org.apache.tika.parser.microsoft.ooxml does not exist
        ParserDecorator.withoutTypes(new org.apache.tika.parser.microsoft.ooxml.OOXMLParser(), EXCLUDES),
                                                                               ^
/Users/--/workplace/opensearch/OpenSearch/plugins/ingest-attachment/src/main/java/org/opensearch/ingest/attachment/TikaImpl.java:101: error: package org.apache.tika.parser.odf does not exist
        new org.apache.tika.parser.odf.OpenDocumentParser(),
                                      ^
/Users/--/workplace/opensearch/OpenSearch/plugins/ingest-attachment/src/main/java/org/opensearch/ingest/attachment/TikaImpl.java:102: error: package org.apache.tika.parser.iwork does not exist
        new org.apache.tika.parser.iwork.IWorkPackageParser(),
                                        ^
/Users/--/workplace/opensearch/OpenSearch/plugins/ingest-attachment/src/main/java/org/opensearch/ingest/attachment/TikaImpl.java:103: error: package org.apache.tika.parser.xml does not exist
        new org.apache.tika.parser.xml.DcXMLParser(),
                                      ^
/Users/--/workplace/opensearch/OpenSearch/plugins/ingest-attachment/src/main/java/org/opensearch/ingest/attachment/TikaImpl.java:104: error: package org.apache.tika.parser.epub does not exist
        new org.apache.tika.parser.epub.EpubParser(), };
                                       ^
```

Reply:
Thank you